chrome/background.js

```diff
@@ -32,6 +32,14 @@
 
 /* global chrome: false */
 
+/* Edge Support - BETA */
+if (typeof msBrowser !== 'undefined') {
+	chrome = msBrowser;
```

Comment:
For ESLint to pass, you'll need to change the comment up top to `/* global chrome: false, msBrowser: false */`, and add `// eslint-disable-line no-native-reassign` at the end of every line that you assign to `chrome`. Same with the other file.

Also some minor code style, see [the other build failures](https://travis-ci.org/honestbleeps/Reddit-Enhancement-Suite/jobs/117013599#L1046).